File: Selecao I/mumia.cpp
/*@mumia: Criança, jovem, adulto*/

#include <iostream>
#include <locale>
using namespace std;

int main() {
    setlocale(LC_ALL, "Portuguese");
    
    string nome;
    int idade;
    
    getline(cin, nome);
    cin >> idade;
    cin.ignore();

    if(idade < 12){
        cout << nome << " eh crianca\n";
    }else if(idade < 18){ 
        cout << nome << " eh jovem\n";
    }else if(idade < 65){
        cout << nome << " eh adulto\n";
    }else if(idade < 1000){
        cout << nome << " eh idoso\n";
    }else{
        cout << nome << " eh mumia\n";
    }
    return 0;
}

File: src/Selecao_II/Mumia.java
/**
 * Questão: @mumia: Criança, jovem, adulto
 *
 * @author UxieGu1
 */

package Selecao_II;
import java.util.Scanner;

public class Mumia {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String nome = scan.nextLine();
        int idade = scan.nextInt();

        if(idade < 12) {
            System.out.println(nome + " eh crianca");
        } else if(idade < 18) {
            System.out.println(nome + " eh jovem");
        } else if(idade < 65) {
            System.out.println(nome + " eh adulto");
        } else if(idade < 1000) {
            System.out.println(nome + " eh idoso");
        } else {
            System.out.println(nome + " eh mumia");
        }

        scan.close();
    }
}
